src/group.ts

```diff
@@ -55,7 +55,7 @@ export abstract class Group<T extends ModuleWorker.Bindings> implements DOG.Grou
 	 */
 	async fetch(input: RequestInfo, init?: RequestInit): Promise<Response> {
 		let request = new Request(input, init);
-		let { pathname } = new URL(request.url, 'foo://');
+		let { pathname } = new URL(request.url);
```

Comment:
This is the only line that had to change. Can you please revert all other changes and make this:

```ts
let { pathname } = new URL(request.url, 'http://dog');
```

That should do the trick

Reply:
I tried, but it is failing with only the change you suggested. The real DOG 'breaks' in the `src/glue.ts` when it tries to make a new request with a non-valid URL here https://github.com/cloudflare/dog/blob/30517cd97bc0911dce8b819296a5cee59ea01ab7/src/glue.ts#L9

Reply:
Ah right, then can update that line too.

```ts
let request = new Request(ROUTES.IDENTIFY, 'http://dog');
```

Reply:
I ended up doing:

```ts
let request = new Request(new URL(ROUTES.IDENTIFY, 'http://dog'));
```

WDYT?

Reply:
oh brain fart 😅 I thought I was typing `new URL` – that's perfect, thanks!